DXMainClient/DXGUI/Generic/UpdateQueryWindow.cs

```diff
@@ -56,11 +56,13 @@ public override void Initialize()
             btnYes.ClientRectangle = new Rectangle(12, 110, 75, 23);
             btnYes.Text = "Yes";
             btnYes.LeftClick += BtnYes_LeftClick;
+            btnYes.Name = "btnYes";
 
             var btnNo = new XNAClientButton(WindowManager);
             btnNo.ClientRectangle = new Rectangle(164, 110, 75, 23);
             btnNo.Text = "No";
             btnNo.LeftClick += BtnNo_LeftClick;
+            btnNo.Name = "btnNo";
```

Comment:
Make use of `nameof`. For example, `btnNo.Name = nameof(btnNo)`.

(I'm aware the old client code doesn't make use of the operator, but that's no reason not to use it in new code.)

Reply:
UpdateQueryWindow has been changed to use this. Will keep in mind for future.